cidr.tf

```diff
@@ -0,0 +1,143 @@
+resource "aws_vpc_ipv4_cidr_block_association" "additional_cidr" {
+  for_each      = toset(var.vpc_additional_cidr)
+  vpc_id        = aws_vpc.main.id
+  cidr_block    = each.value
+}
+
+# Additional Private Subnets #
+# PRIVATE A[count+2]
```

Comment:
Could we remove this comment. It is misleading?

Reply:
Comment was removed.